terraform/environments/electronic-monitoring-data/s3.tf

```diff
@@ -0,0 +1,56 @@
+#------------------------------------------------------------------------------
+# S3 bucket for bucket action logs
+#------------------------------------------------------------------------------
+
+resource "aws_s3_bucket" "log_bucket" {
+  bucket_prefix = "em-data-store-logs-"
+}
+
+#resource "aws_s3_bucket_acl" "log_bucket_acl" {
+#  bucket = aws_s3_bucket.log_bucket.id
+#  acl    = "log-delivery-write"
+#}
+
+#------------------------------------------------------------------------------
+# S3 bucket for landing Capita data
+#
+# Lifecycle management not implemented for this bucket as everything will be
+# moved to a different bucket once landed.
+#------------------------------------------------------------------------------
+
+resource "random_string" "capita_random_string" {
+  length  = 10
+  lower   = true
+  upper   = false
+  numeric = true
+  special = false
+}
+
+resource "aws_s3_bucket" "capita_landing_bucket" {
+  bucket = "capita-${random_string.capita_random_string.result}"
```

Comment:
Not going to block on this, but as we only want one of these buckets per supplier per environment, whether we'd be better off either hard-coding a bucket name, or parameterising the bucket name via environment etc. I feel like adding randomisation could easily have unintended effects if for some reason terraform has recreates the `random_string.capita_random_string` resource.

Reply:
One of the recommendation's from cyber security was to have a randomised bucket name. Given the bucket is only going to be used as a landing point before the data is moved to an internal bucket (using [`aws_transfer_workflow`s](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transfer_workflow)) we can refer to the bucket by its id throughout the terraform.

Reply:
Is there a document/diagram somewhere that describes the bucket chain that you describe?